stdlib/3/io.pyi

```diff
@@ -1,150 +1,211 @@
 # Stubs for io
 
-# Based on http://docs.python.org/3.2/library/io.html
-
-from typing import List, BinaryIO, TextIO, IO, overload, Iterator, Iterable, Any
+from typing import (
+    List, BinaryIO, TextIO, Iterator, Union, Optional, Callable, Tuple
+)
 import builtins
 import codecs
-import _io
+import sys
+from types import TracebackType
+
+DEFAULT_BUFFER_SIZE = ...  # type: int
 
-DEFAULT_BUFFER_SIZE = 0  # type: int
-SEEK_SET = ...  # type: int
-SEEK_CUR = ...  # type: int
-SEEK_END = ...  # type: int
+if sys.version_info >= (3, 1):
```

Comment:
Mypy doesn't support Python 3.0 or 3.1, so you can put this block in unconditionally.